interfaces/ldconfig/backend.go

```diff
@@ -52,6 +53,12 @@ func (b *Backend) Name() interfaces.SecuritySystem {
 // If the method fails it should be re-tried (with a sensible strategy) by the caller.
 func (b *Backend) Setup(appSet *interfaces.SnapAppSet, opts interfaces.ConfinementOptions, repo *interfaces.Repository, tm timings.Measurer) error {
 	snapName := appSet.InstanceName()
+	if !interfaces.IsTheSystemSnap(snapName) {
```

Comment:
this might be ok for now but we probably want to instead change the rest of the code to work even if this is not here

Reply:
With the change in the checks in the specification, this check is not needed anymore. But that is sort of by chance as the spec is empty, if it wasn't it would try to write in `/etc/ld.so.conf.d/snap.system.conf` for a snap that is not the system snap. So I think it is better to leave for the moment just to make clear we do not support normal snaps for the moment.

Reply:
but there two sides to things, you need to be abel to update things both when the system snap is update, but also when the providing snap with the slot which a kernel or something is updated, so I don't think we can really have this check

Reply:
changed now